packages/vanilla/src/sass/layout/_container.scss

```diff
@@ -1,8 +1,8 @@
 @use './../tokens/breakpoints' as *;
 
 .cbp-fluid-container {
-  padding-left: var(--cbp-space-4x);
-  padding-right: var(--cbp-space-4x);
+  padding-left: var(--cbp-space-5x);
+  padding-right: var(--cbp-space-5x);
```

Comment:
20px seems like a very large amount of padding for a potentially small device. If this is the spec, I think we need to discuss it further. I typically see very little padding at the mobile sizes and use .5rem tops myself. 40px is 10% (or more) of the screen width on a phone.